controllers/extension-certificate-service/pkg/controller/config/config.go

```diff
@@ -0,0 +1,22 @@
+// Copyright (c) 2019 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package controller
+
+import "github.com/gardener/gardener-extensions/controllers/extension-certificate-service/pkg/apis/config/v1alpha1"
+
+// Config contains configuration for the certificate service.
+type Config struct {
```

Comment:
Why do you have this?

Reply:
It's needed to stick to the controller construction process we have on the top: https://github.com/gardener/gardener-extensions/blob/ea9fca24d90bd0f6f6c5917755c55780b39a6780/controllers/extension-certificate-service/cmd/app/app.go#L69

`Complete()` -> `Completed()` -> `Apply()` pattern.

Concretely, to `Apply` the parsed configuration from the command line to the configuration used by the reconcilers.

Reply:
OK, thank you!